src/order/basic.lean

```diff
@@ -153,6 +143,16 @@ lemma monotone.ne_of_lt_of_lt_int {α} [preorder α] {f : ℤ → α} (hf : mono
   (x x' : ℤ) {y : α} (h1 : f x < y) (h2 : y < f (x + 1)) : f x' ≠ y :=
 by { rintro rfl, apply (hf.reflect_lt h1).not_le, exact int.le_of_lt_add_one (hf.reflect_lt h2) }
 
+theorem monotone_const {β} [preorder β] {b : β} : monotone (λ(a:α), b) := assume x y h, le_refl b
+
+lemma monotone_of_monotone_nat {α} [preorder α] {f : ℕ → α} (hf : ∀n, f n ≤ f (n + 1)) :
+  monotone f | n m h :=
+begin
+  induction h,
+  { refl },
+  { transitivity, assumption, exact hf _ }
+end
+
```

Comment:
Why did these move?

Reply:
I thought that they belonged closer to the other monotone lemmas, now that the assumptions are more similar...  I can move them back, if you think it is better.

Reply:
Did any of the assumptions change?

Reply:
Initially, I thought that you were referring to a different set of lemmas.  These I had moved since I tried to weaken the assumptions of `monotone` to `has_le`, instead of `preorder`.  For that change, I replaced `preorder` by `has_le` in this commit as well (085b1720d9137850ccca940e449f907cb7aa3227).  Since now `refl` was not available, I moved the lemmas to later, when the preorder assumption became available.  Once I reverted the change to `monotone` because of a failing `apply_fun` test, I forgot to move these lemmas back to their place.

There is no need for them to be here: they can move back to where they were!

Reply:
I just pushed a version where these lemmas have moved back.  I also realized that I had changed their assumptions "in-line", not because they were in a section with different assumptions and I undid that change as well.